bindings/java/src/test/java/org/apache/opendal/operator/FileSystemOperatorTest.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.opendal.operator;
+
+
+import org.apache.opendal.FileSystemOperator;
+import org.apache.opendal.Operator;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileSystemOperatorTest {
+
+    @Test
+    public void testBuilder() {
+
+        Operator op = new FileSystemOperator("/tmp");
```

Comment:
I expect to see something like `Operator("fs", "/tmp")` instead of different operators for different services.